### PR TITLE
Columns toggler affix

### LINKED ISSUE #137: Affix for table headers/columns toggler
Currently, table has no pagination. If we want to support infinite scrolling (and I think we can make that work) we should make sure user has access to columns toggler/table headers no matter how far down they scrolled.

### PR DESCRIPTION
**Pull request by smogg**
Builds on top of #140 
Closes #137

### CODE REVIEW

**Comment by lvh** [comment]
Could you please merge this forward?

**Comment by smogg** [comment]
Proszę bardzo.

File: src/desdemona/ui/table.cljs [hunk 1]
```diff
@@ -14,7 +14,19 @@
   ((if (contains? coll k)
      disj conj) coll k))
 
-(defn columns-toggler-component
+(defn make-toggler-affix
+  []
+  (let [toggler (sel1 :#columns-toggler)
+        page-nav (sel1 :#page-nav)
+        page-scroll-y (.-pageYOffset js/window)]
+    (when toggler
```
Comment (lvh) [inline src/desdemona/ui/table.cljs]: When is there no toggler?

File: src/desdemona/ui/table.cljs [hunk 2]
```diff
@@ -14,7 +14,19 @@
   ((if (contains? coll k)
      disj conj) coll k))
 
-(defn columns-toggler-component
+(defn make-toggler-affix
+  []
+  (let [toggler (sel1 :#columns-toggler)
```
Comment (lvh) [inline src/desdemona/ui/table.cljs]: Does this mean that there can only ever be exactly 1 such table?

**Comment by lvh** [comment]
What does the performance look like for this?

File: src/desdemona/ui/table.cljs (hunk 1 above)
Comment (smogg) [inline src/desdemona/ui/table.cljs]: When it's a different tab, e.g: dashboard

File: src/desdemona/ui/table.cljs (hunk 2 above)
Comment (smogg) [inline src/desdemona/ui/table.cljs]: At this point, yeah. We might make it more reusable in the future as needed (but I'm assuming you'd rather have me do that now?)

**Comment by smogg** [comment]
It runs smoothly for me in Firefox. There are some delays when using requestAF/next-tick in chrome/safari while scrolling, but I"m not sure how to solve that yet. It is _usable_ tho

File: src/desdemona/ui/table.cljs (hunk 2 above)
Comment (lvh) [inline src/desdemona/ui/table.cljs]: Well, that poses an interesting question: I think there should be more than 1 such table, but then toggler affixing becomes trickier: either you affix togglers to all of them at once (which doesn't sound right) or you still have to look at 1 table at a time. I think it should be resuable that way now, but I don't think it's a hard requirement, so if it's a ton of work it's OK to skip it.

**Comment by lvh** [comment]
This LGTM unless you want to do the multi-table stuff now.

**Comment by smogg** [comment]
Let's leave it for now - I'll finish other issues and come back

**Pull request merged**